apps/api/package.json

```diff
@@ -62,31 +61,24 @@
     "swagger-jsdoc": "^6.2.8",
     "swagger-ui-express": "^5.0.1",
     "ts-node": "^10.9.1",
-    "typescript-eslint": "^8.23.0",
     "uuid": "^9.0.0"
   },
   "devDependencies": {
     "@elizaos/core": "^0.25.9",
-    "@eslint/js": "^9.19.0",
+    "@recallnet/eslint-config": "workspace:^",
+    "@recallnet/typescript-config": "workspace:*",
     "@types/bcrypt": "^5.0.0",
     "@types/cors": "^2.8.13",
     "@types/jest": "^29.5.0",
     "@types/pg": "^8.6.6",
     "@types/swagger-jsdoc": "^6.0.4",
     "@types/swagger-ui-express": "^4.1.8",
     "@types/uuid": "^9.0.1",
-    "@typescript-eslint/eslint-plugin": "^8.23.0",
-    "@typescript-eslint/parser": "^8.23.0",
-    "@typescript-eslint/utils": "^8.23.0",
     "axios": "^1.6.0",
-    "eslint": "^9.19.0",
-    "eslint-config-prettier": "^10.0.1",
-    "eslint-plugin-prettier": "^5.2.3",
     "jest": "^29.5.0",
     "openapi-markdown": "^1.2.3",
-    "prettier": "^3.4.2",
     "ts-jest": "^29.1.0",
     "ts-node-dev": "^2.0.0",
-    "typescript": "^5.0.0"
```

Comment:
Remove deps that are provided by the shared configs we now depend on, or things that are managed elsewhere like prettier.